db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170319214240) do
+ActiveRecord::Schema.define(version: 20170613071826) do
```

Comment:
Yeah, each table should have one single migration file.

Methods `up/down` are actually combined in one method `change`.

`up/down` are called the old migration methods, you can achieve the same intended behaviour using `change` http://guides.rubyonrails.org/active_record_migrations.html#using-the-change-method